src/Honeycomb.OpenTelemetry/HoneycombOptions.cs

```diff
@@ -76,6 +50,15 @@ static HoneycombOptions()
         /// </summary>
         public string ApiKey { get; set; }
 
+        /// <summary>
+        /// Confirm whether api key is legacy
+        /// </summary>
+        public Boolean isLegacyKey()

```

Comment:
.NET uses PascalCase for method named.
```suggestion
        public Boolean IsLegacyKey()
```